api/logic/Explanation.java

```diff
@@ -23,17 +23,14 @@
 
 import com.vaticle.typedb.client.api.answer.ConceptMap;
 
-import java.util.Map;
 import java.util.Set;
 
 public interface Explanation {
-
     Rule rule();
 
     ConceptMap conclusion();
 
     ConceptMap condition();
 
-    Map<String, Set<String>> variableMapping();
-
+    Set<String> getMapping(String var);
```

Comment:
This isn't clear what the key `var` is going to get - is it a variable from the `condition` or the `conclusion`? Maybe we should update the naming?